utils/utils.py

```diff
@@ -158,3 +157,27 @@ def product_list(num_list):
     for dim in num_list:
         product *= dim
     return product
+
+def mkdir(directory):
+    """ Create directory if it does not exist. """
+    if not os.path.exists(directory):
+        try:
+            os.makedirs(directory)
+        except OSError:
+            print("Directory {} already exists.".format(directory))
```

Comment:
surely this except will never get hit with the if statement on 163?

Reply:
Probably true, I'll test and just remove the if and keep the try-except